mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
Just define two managers `Work`, see https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset

NB: Since we use a custom QuerySet for Works, you should use `from_queryset`; something like the following:

```python
class FilteredWorkManager(models.Manager):
    def get_queryset(self):
        return super().get_queryset().filter(redirect__isnull=True)

class Work(models.Model):
    ...
    objects = FilteredWorkManager.from_queryset(WorkQuerySet)()
    all_objects = WorkQuerySet.as_manager()
```